Tests/OctopusDSC.Tests.ps1

```diff
@@ -84,139 +99,120 @@ Describe "OctopusDSC.psd1" {
 
         $expected = ($Data.DscResourcesToExport | Sort-Object) -join ","
         $actual = (($modules.Name | Sort-Object) -join ",")
-        $actual | should be $expected
+        $actual | Should -Be $expected
     }
 }
 
 Describe "Mandatory Parameters" {
+    . (Join-Path -Path $script:scriptRoot -ChildPath "powershell-helpers.ps1")
```

Comment:
Lots of changes here - we were creating tests inside a foreach, but that's no longer working.
We now need to setup the test cases and have a single `it` block to execute it.